app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesScreen.kt

```diff
@@ -83,3 +100,86 @@ fun NoUpdatesScreen() {
     )
   }
 }
+
+@Composable
+private fun AppsList(
+  appList: List<App>,
+  navigate: (String) -> Unit,
+) {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth(),
+    horizontalAlignment = Alignment.CenterHorizontally
+  ) {
+    UpdateBox()
+    LazyColumn(
+      modifier = Modifier
+        .semantics { collectionInfo = CollectionInfo(appList.size, 1) }
+        .padding(start = 16.dp, end = 16.dp)
+        .wrapContentSize(Alignment.TopCenter)
+    ) {
+      itemsIndexed(appList) { index, app ->
+        AppItem(
+          app = app,
+          onClick = {
+            navigate(
+              buildAppViewRoute(app)
+                .withItemPosition(index)
+            )
+          },
+        ) {
+          InstallViewShort(app)
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun UpdateBox() {
+  Box(
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(start = 16.dp, end = 16.dp, top = 16.dp, bottom = 32.dp)
```

Comment:
Use 16dp bottom padding for now, since each app item currently already has 16dp of top padding.